src/custom-elements/src/index.html

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
-<html dir="ltr" lang="en">
+<html dir="ltr" lang="en" class="chi">
```

Comment:
This web components should be able to run without chi configured in the page. This might hide errors in the components' css that will be automatically be fixed by this class an the chi.css file.